asnets/asnets/interfaces/fd_interface.py

```diff
@@ -194,7 +194,7 @@ def make_wlama_args(w):
         # fast-downward.py --alias lama-first ${dom} ${prob}
         cmdline.extend([
             domain_bn, problem_bn, "--evaluator",
-            ("hlm=lmcount(lm_factory=lm_rhw(reasonable_orders=true),"
+            ("hlm=landmark_sum(lm_factory=lm_rhw(),"
```

Comment:
```suggestion
            ("hlm=landmark_sum(lm_factory=lm_reasonable_orders_hps(lm_rhw()),"
```
This preserves the reasonable_orders=true flag. Please check that the planner call still works.

Reply:
Thanks for the suggestion : )